app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <!-- Internet Permission -->
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
I'm getting a warning that this permission is deprecated.

<img width="666" alt="image" src="https://github.com/feedme-swent-epfl/feedme-android/assets/100317488/eb3580d2-90ba-4244-b609-43ad39d35192">